controllers/databaseclusterbackup_controller.go

```diff
@@ -115,6 +115,11 @@ func (r *DatabaseClusterBackupReconciler) Reconcile(ctx context.Context, req ctr
 		}
 		return reconcile.Result{}, err
 	}
+
+	if backup.IsComplete() && backup.GetDeletionTimestamp().IsZero() {
+		return reconcile.Result{}, err
```

Comment:
We don't have an `err` to return here

Reply:
Actually it makes no difference but I'd rather us be explicit.
```suggestion
		return reconcile.Result{}, nil
```